Version 0.17.12 makes things no longer Send

Ideally, the version should be retracted since it stops code from building after updating from .11 to .12.

(Also, there is no tag for .12 in the repo)

My build error is the following

```
error[E0277]: `(dyn for<'a, 'b, 'c, 'd> Fn(&'a [u8], &'b mut [u8], &'c png::common::Info<'d>) + 'static)` cannot be sent between threads safely
   --> loaders/glycin-image-rs/src/bin/glycin-image-rs.rs:116:31
    |
116 | impl LoaderImplementation for ImgDecoder {
    |                               ^^^^^^^^^^ `(dyn for<'a, 'b, 'c, 'd> Fn(&'a [u8], &'b mut [u8], &'c png::common::Info<'d>) + 'static)` cannot be sent between threads safely
    |
    = help: the trait `Send` is not implemented for `(dyn for<'a, 'b, 'c, 'd> Fn(&'a [u8], &'b mut [u8], &'c png::common::Info<'d>) + 'static)`
    = note: required for `Unique<(dyn for<'a, 'b, 'c, 'd> Fn(&'a [u8], &'b mut [u8], &'c png::common::Info<'d>) + 'static)>` to implement `Send`
note: required because it appears within the type `Box<dyn Fn(&[u8], &mut [u8], &Info<'d>)>`
   --> /home/herold/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/boxed.rs:195:12
    |
195 | pub struct Box<
    |            ^^^
note: required because it appears within the type `Option<Box<dyn Fn(&[u8], &mut [u8], &Info<'d>)>>`
   --> /home/herold/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/option.rs:569:10
    |
569 | pub enum Option<T> {
    |          ^^^^^^
note: required because it appears within the type `Reader<Cursor<Vec<u8>>>`
   --> /home/herold/.cargo/registry/src/index.crates.io-6f17d22bba15001f/png-0.17.12/src/decoder/mod.rs:356:12
    |
356 | pub struct Reader<R: Read> {
    |            ^^^^^^
note: required because it appears within the type `PngDecoder<Cursor<Vec<u8>>>`
   --> /home/herold/.cargo/registry/src/index.crates.io-6f17d22bba15001f/image-0.24.8/src/codecs/png.rs:112:12
    |
112 | pub struct PngDecoder<R: Read> {
    |            ^^^^^^^^^^
note: required because it appears within the type `ImageRsDecoder<Cursor<Vec<u8>>>`
   --> loaders/glycin-image-rs/src/bin/glycin-image-rs.rs:158:10
   ```

--- a/src/decoder/transform.rs
+++ b/src/decoder/transform.rs
@@ -12,7 +12,7 @@ use super::stream::FormatErrorInner;
 ///
 /// TODO: If some precomputed state is needed (e.g. to make `expand_paletted...`
 /// faster) then consider changing this into `Box<dyn Fn(...)>`.
-pub type TransformFn = Box<dyn Fn(&[u8], &mut [u8], &Info)>;
+pub(crate) type TransformFn = Box<dyn Fn(&[u8], &mut [u8], &Info) + Send + Sync>;
 
 /// Returns a transformation function that should be applied to image rows based
 /// on 1) decoded image metadata (`info`) and 2) the transformations requested

--- a/src/decoder/transform.rs
+++ b/src/decoder/transform.rs
@@ -12,7 +12,7 @@ use super::stream::FormatErrorInner;
 ///
 /// TODO: If some precomputed state is needed (e.g. to make `expand_paletted...`
 /// faster) then consider changing this into `Box<dyn Fn(...)>`.
-pub(crate) type TransformFn = Box<dyn Fn(&[u8], &mut [u8], &Info) + Send + Sync>;
+pub type TransformFn = Box<dyn Fn(&[u8], &mut [u8], &Info) + Send + Sync>;
 
 /// Returns a transformation function that should be applied to image rows based
 /// on 1) decoded image metadata (`info`) and 2) the transformations requested

